microsoft/botframework-solutions
# [Generator][TypeScript] Implement Code Coverage

## Description
- Implement `nyc` for **Code Coverage**
- Add `.nycrc` file
- Update `mocha.opts`
- Improvement in `after` method in generators
- Update `.npmignore`
- Add `.gitignore`

## Testing Steps

1.  Go to `AI\templates\Virtual-Assistant-Template\typescript\generator-botbuilder-assistant`
1. Run `npm install`
1. Run `npm run coverage`
1. Check the following output after executing the command
![image](https://user-images.githubusercontent.com/11904023/57784951-2d4ce000-7707-11e9-855d-361e3d8aa161.png)
